### ISSUE TITLE
metabolomics only samples

### ISSUE THREAD

**Issue by ssarrafan** [microbiomedata/nmdc-schema#624]
Metadata ingest for metabolomics only samples for bioscales

**Comment by turbomam** [comment]
@ssarrafan Is this a request for a change in the schema?

**Comment by aclum** [comment]
No, this is to import bioscales metabolomics only samples into NMDC. No new schema changes are required for that. Would you prefer this ticket be in a different repo?

**Comment by aclum** [comment]
We need an envio triad for these bioscales leaf material samples. @emileyfadrosh  @mslarae13 @cmungall 

To be discussed at the planning meeting on Monday. What metadata should be harvested from GOLD? We can use the same field research site (tree name) to pull in metadata from GOLD.

**Comment by turbomam** [comment]
This issue does not belong in the nmdc-schema repo because it doe not request changes to the schema. It might make sense to put it in sample-annotator.
